Feature: Lazy unmount (#1705)
Azure/azure-storage-fuse: CHANGELOG.md

## 2.5.0~preview.1 (Unreleased)
**Features**
- Preload feature added to download entire dataset on mount, to accelerate model training.

> - Added `lazy` flag to unmount command which passes the lazy flag to fusermount when unmounting.

Reviewer: ```suggestion
-Added support for lazy unmounts. If device is busy when unmount was executed, it will wait for device to be free and unmount automatically.
```

> - Added support for lazy unmounts. If device is busy when unmount was executed, it will wait for device to be free and unmount automatically.

Reviewer: ```suggestion
- Added support for lazy unmounts. Lazy unmount will wait for device to be free and unmount automatically, instead of giving "device or resource busy" on executing unmount.
```

> - Added support for lazy unmounts. Lazy unmount will wait for device to be free and unmount automatically, instead of giving "device or resource busy" on executing unmount.

Reviewer 2: Add a comment how to enable lazy mount i.e the flag
